tests/time_tests/test_runner/utils.py

```diff
@@ -58,6 +58,49 @@ def metadata_from_manifest(manifest: Path):
     }
 
 
+# -------- Platform specific --------
+
+def get_platform_info(platform_alias):
```

Comment:
We should avoid using code names in public code. Code names usually require usage approval or might have embargoed usage period. Also, names mapping seems a GUI/UI feature and should be solved there.

Reply:
Decided to close PR in order to do mapping in another way